packages/react-ui/src/theme/sections/typography.js

```diff
@@ -9,117 +11,219 @@ export const themeTypography = {
+    fontWeight: 600
+  },
+  body1StrongItalic: {
+    ...baseTypography.body1,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  body2Italic: {
+    ...baseTypography.body2,
+    fontStyle: 'italic'
+  },
+  body2Strong: {
+    ...baseTypography.body2,
+    fontWeight: 600
+  },
+  body2StrongItalic: {
+    ...baseTypography.body2,
+    fontWeight: 600,
+    fontStyle: 'italic'
+  },
+  buttonLarge: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: '0.25px'
+  },
+  buttonSmall: {
+    fontFamily: 'Inter, sans-serif',
+    fontWeight: 500,
+    fontSize: getPixelToRem(11),
+    lineHeight: 1.454,
+    letterSpacing: '0.4px'
+  },
+  captionItalic: {
+    ...baseTypography.caption,
+    fontStyle: 'italic'
+  },
+  captionStrong: {
+    ...baseTypography.caption,
+    fontWeight: 500
+  },
+  captionStrongItalic: {
+    ...baseTypography.caption,
+    fontWeight: 500,
+    fontStyle: 'italic'
+  },
+  overlineDelicate: {
+    ...baseTypography.overline,
+    fontWeight: 400,
+    letterSpacing: '1.2px'
+  },
+  codeLarge: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 400,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeLargeStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(16),
+    lineHeight: 1.5,
+    letterSpacing: 0
+  },
+  codeMedium: {
+    fontFamily: '"Overpass Mono", monospace',
     fontWeight: 400,
-    fontSize: '0.625rem',
-    lineHeight: 1.6,
-    letterSpacing: '0.150em',
-    textTransform: 'uppercase',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(14),
+    lineHeight: 1.428,
+    letterSpacing: 0
+  },
+  codeMediumStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(14),
+    lineHeight: 1.428,
+    letterSpacing: 0
+  },
+  codeSmall: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 400,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.333,
+    letterSpacing: 0
+  },
+  codeSmallStrong: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.333,
+    letterSpacing: 0
+  },
+  codeSmallChip: {
+    fontFamily: '"Overpass Mono", monospace',
+    fontWeight: 600,
+    fontSize: getPixelToRem(12),
+    lineHeight: 1.25,
+    letterSpacing: 0
   },
   charts: {
-    fontFamily: '"Open Sans", sans-serif',
+    // TODO: design team is going to create a new variant (more reusable) to replace this one
+    fontFamily: 'Inter, sans-serif',
     fontWeight: 400,
-    fontSize: 10,
-    lineHeight: 1.6,
-    letterSpacing: '0.150em',
-    fontSmoothing: 'antialiased'
+    fontSize: getPixelToRem(10),
+    lineHeight: 1.2,
+    letterSpacing: '2px'
   }
 };
+
+export const themeTypography = {
+  ...baseTypography,
+  ...customTypography
```

Comment:
I'm not sure about including both blocks at the same level... but probably we can move on and have a longer discussion